### PR TITLE
Tickets/sitcom 1639

### PR DESCRIPTION
**Pull request by isotuela**
This is a minor restructuring of Simonyi, AuxTel and Observatory Control System folders to reflect more the OBS division System/Subsystem. (although there's a lot of moved files) I have also taken advantage to add some of the information required in SITCOM-1703. More details and links will be added in the near future. 

Thank you for your review. 
Ioana

### CODE REVIEW

File: AuxTel/Components/EUI-Access/index.rst
Comment (JackieS-NL) [inline AuxTel/Components/EUI-Access/index.rst:1]: This was under Non-standard procedures before, I think it’s better suited in there.

File: AuxTel/Non-Standard-Operations/index.rst
Comment (JackieS-NL) [inline AuxTel/Non-Standard-Operations/index.rst:1]: “AuxTel E-stop procedure” is repeated and there is “<<<<<<< HEAD” before.

File: Simonyi/index.rst [hunk 1]
```diff
@@ -41,8 +41,7 @@ Introduction to the Simonyi Telescope
 
    Components/MTCS/index.rst
    Components/*
-   Components/Auxiliary-Components/index.rst
-
+   
 .. _Simonyi-Telescope-Weather-Constraints:
 
 Weather Constraints
```
Comment (JackieS-NL) [inline Simonyi/index.rst:47]: There are 2 links that take to the same page "Simonyi Weather Constraints"

**Review by JackieS-NL — CHANGES_REQUESTED** [review]
I like this structure, thank you! 
I added 3 comments. 
A fourth comment is that for both AuxTel and Simonyi “Calibration system” page is repeated, there is one under components and another under troubleshooting.

File: AuxTel/Non-Standard-Operations/index.rst
Comment (isotuela) [reply AuxTel/Non-Standard-Operations/index.rst:1]: Thanks! Not sure what happened in the rebase there.

File: Simonyi/index.rst (hunk 1 above)
Comment (isotuela) [reply Simonyi/index.rst:47]: Thank you, I have add the option `:hidden:` to the `toctree` so it shows in the left hand side menu, but not in the page itself.

**Comment by isotuela** [comment]
@JackieS-NL Thank you for your review! 
Re your fourth comment, I have decided to keep the same name of the pages for now because they belong to different sections. The menus would look too heavy (in my opinion) with all the "ATCalSys Troubleshooting", "ATCS Non Standard Operations". I am still looking for a better way, but will leave it as it is until we find a better solution. 
Cheers, 
Ioana

**Review by JackieS-NL — APPROVED** [review]
Thanks Ioana

**Comment by isotuela** [comment]
Many thanks, @JackieS-NL!

**Comment by kpenaramirez** [comment]
Sorry for the late reaction. I just compared the obs-doc versions and the changes definitely go in the right direction. Thank you both, @isotuela and @JackieS-NL .

**Pull request merged**